### ISSUE TITLE
Implement TurboShark Webcam Capture Node

### ISSUE THREAD

**Issue by mvollrath** [EndPointCorp/lg_ros_nodes#281]
### Overview
We want to capture a webcam video stream and pipe it to a [Janus Gateway](https://github.com/meetecho/janus-gateway) instance for remote viewing.

### Background
[Serve Webcam Stream for Monitoring](https://github.com/EndPointCorp/lg_chef/issues/934)
[TurboShark](https://github.com/EndPointCorp/lg_chef/issues/884)

### Requirements
* ROS script that launches GStreamer with parameters for path to video device, output width/height/fps, target host, and target port.
* Use realtime VP8 encoding, RTP/UDP transport.

### Development notes
A GStreamer pipeline used for PoC:

`gst-launch-1.0 v4l2src device=/dev/video0 ! videoscale ! capsfilter caps=video/x-raw,width=640,height=360,framerate=60/1 ! queue ! vp8enc target-bitrate=768000 deadline=33333 cpu-used=16 ! rtpvp8pay ! udpsink sync=false qos=true host=X.X.X.X port=NNNN`

Basically, fill in the parameters and run this pipeline.

**Comment by wzin** [comment]
it spawns gstreamer as expected. closing.
